src/Button.php

```diff
@@ -57,6 +56,44 @@ public static function link(
         return self::widget($constructorArguments, $config, $theme)->label($label)->type(ButtonType::LINK)->url($url);
     }
 
+    /**
+     * Get an instance of a reset button input.
+     *
+     * @param string|Stringable $value The content of the button. By default, it's "Reset".
+     * @param array $constructorArguments The constructor arguments.
+     * @param array $config The configuration.
+     * @param string|null $theme The theme.
+     *
+     * @return self A new instance with the input of "reset" type.
+     */
+    public static function resetInput(
+        string|Stringable $value = 'Reset',
+        array $constructorArguments = [],
+        array $config = [],
+        string|null $theme = null
+    ): self {
+        return self::widget($constructorArguments, $config, $theme)->label($value)->type(ButtonType::INPUT_RESET);
+    }
+
+    /**
+     * Get an instance of a submit button input.
+     *
+     * @param string|Stringable $value The content of the button. By default, it's "Submit".
+     * @param array $constructorArguments The constructor arguments.
+     * @param array $config The configuration.
+     * @param string|null $theme The theme.
+     *
+     * @return self A new instance of an input with "submit" type.
+     */
+    public static function submitInput(
```

Comment:
For consistency factory methods and enum cases should be equal. Currently, these are different: `submitInput()` and `INPUT_SUBMIT`.

Reply:
done.